src/Agent.Worker/Handlers/Helpers/ProcessHandlerHelper.cs

```diff
@@ -1,66 +1,46 @@
-﻿using System;
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
 using System.Collections.Generic;
-using System.Text;
+using Agent.Sdk.Knob;
+using Microsoft.VisualStudio.Services.Agent.Util;
+using Microsoft.VisualStudio.Services.Agent.Worker;
+using Microsoft.VisualStudio.Services.Common;
 
 namespace Agent.Worker.Handlers.Helpers
 {
     public static class ProcessHandlerHelper
     {
-        public static (string, CmdTelemetry) ProcessInputArguments(string inputArgs)
+        private const char _EscapingSymbol = '^';
+        private const string _EnvPrefix = "%";
+        private const string _EnvPostfix = "%";
```

Comment:
C# convention is camelCase.  underscore is okay

Reply:
yes, my doubt was that const are CamelCase but private are _camelCase. So, I just combined the private + const conventions 😄 
If private const also _camelCase it's ok for me to update

Reply:
updated